test/Mutex/MutexTest.php

```diff
@@ -5,6 +5,8 @@
 use Amp\Delayed;
 use Amp\Redis\Config;
 use Amp\Redis\IntegrationTest;
+use Amp\Redis\QueryExecutor;
+use Amp\Redis\QueryExecutorFactory;
```

Comment:
Accidental add? Or forgot to commit added tests?

Reply:
Yes, wrote a test, then noticed it doesn't cover what I actually wanted to test.